Transactions.md

```diff
@@ -30,9 +30,12 @@ The funding inputs and change output script public keys are negotiated in the of
 
 ## Funding Transaction Input and Output Ordering
 
-The Offerer/Initiator's inputs come first and in the order that they were listed in the Offer message, the Accepter's inputs come after this in the order they are listed in the Accept message. The funding output comes first, followed by the Offerer's change output if applicable, and then the Accepter's change output if applicable.
+The inputs are sorted by each `funding_input`'s `input_serial_id` in ascending order.
+The outputs are sorted by `fund_output_serial_id` in ascending order with the funding output having an `fund_output_serial_id` of `0x8000000000000000`
 
-Note that this will likely change in the future.
+The funding output has a `fund_output_serial_id` of `0x8000000000000000` since it is the midpoint of an uint64.
+This is chosen to give the funding output so each user's output has an equal likelihood of being before or after the funding output.
```

Comment:
I'm still confused why this is here? Why not just have a random number like the rest? The dual funded lightning channel proposal gives the funding output no special treatment.

The issue here is that this introduces a fingerprint as the funding output is more likely to be near the middle instead of in a random place.

Reply:
Tried to be clever thinking this was an equal distribution, ran some simulations and now see that it does favor having the funding output in the middle.

Reply:
Updated to give it its own serial id